modules/cnv-about-dedicated-resources.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_advanced_vm_management/cnv-dedicated-resources-vm.adoc
+// * cnv/cnv_vm_templates/cnv-dedicated-resources-vm-template.adoc
+
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+[id="cnv-about-dedicated-resources_{context}"]
+
+= About dedicated resources
+
+When you enable dedicated resources for your virtual machine, your virtual
+machine's workload is scheduled on CPUs that will not be used by other
+processes. By using dedicated resources, you improve the performance of the
+virtual machine and the accuracy of latency predictions.
+
+== Labels, nodes, and workload scheduling
+
+To enable dedicated resources, the `cpumanager` policy on the node
```

Comment:
I wonder if there is adequate information here on how to do this.

Reply:
@matthewcarleton I've added more info and restructured, worded differently. Let me know if you think more info needs to be provided.

Putting this to QE and final Peer Review.